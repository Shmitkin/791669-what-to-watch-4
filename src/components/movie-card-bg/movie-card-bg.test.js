import React from "react";
import renderer from "react-test-renderer";
import MovieCardBg from "./movie-card-bg.jsx";

it(`Should MovieCardBg render correctly`, () => {
  const tree = renderer
    .create(
        <MovieCardBg altDesc = {`Some bg Description`}/>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
