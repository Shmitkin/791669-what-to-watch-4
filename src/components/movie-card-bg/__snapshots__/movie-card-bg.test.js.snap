// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieCardBg render correctly 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="Some bg Description"
    src="img/bg-the-grand-budapest-hotel.jpg"
  />
</div>
`;
